sync3/handler/connstate_live.go

```diff
@@ -187,6 +187,14 @@ func (s *connStateLive) processLiveUpdate(ctx context.Context, up caches.Update,
 		// include this update in the rooms response TODO: filters on event type?
 		userRoomData := roomUpdate.UserRoomMetadata()
 		r := response.Rooms[roomUpdate.RoomID()]
+
+		conMeta := s.lists.ReadOnlyRoom(roomUpdate.RoomID())
+		for _, ts := range conMeta.LastInterestedEventTimestamps {
```

Comment:
Needs comments. This is using the highest timestamp from each list for this room I guess?